webhooks/framework.go

```diff
@@ -0,0 +1,55 @@
+package webhooks
+
+import (
+	"fmt"
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/rancher/webhook-service/drivers"
+)
+
+// ConstructPayload constructs jwt url
+func ConstructPayload(input map[string]interface{}) (map[string]interface{}, error) {
+	if _, ok := input["driver"].(string); !ok {
+		return nil, fmt.Errorf("Driver of type string not provided\n")
+	}
+	driverId := input["driver"].(string)
```

Comment:
Don't do these naked type conversions. Add an `ok` check to them so that they can't cause a panic.